Yoast/javascript
# Create rules to stem superlatives

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds functionality to stem Spanish superlatives.

## Relevant technical choices:

* I removed a test for "friísimo" from the list of superlatives to test - this superlative will be covered by fullForm exceptions rule, which is not implemented yet. 


## Test instructions

This PR can be tested by following these steps:

* Check that the spec contains all possible types of superlatives.
* Run `yarn test` and check if it passes.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-275
